dashboard/views.py

```diff
@@ -0,0 +1,20 @@
+from django.core.exceptions import BadRequest
+from django.http import HttpResponse
+from django.shortcuts import render
+
+from .service import generate_embed_url_for_anonymous_user
+
+
+def metadata_quality_dashboard(request) -> HttpResponse:
+    quicksight_embedded_url = generate_embed_url_for_anonymous_user()
+    if not quicksight_embedded_url:
+        raise BadRequest("Error generating QuickSight embedded URL")
```

Comment:
This seems like the wrong use of BadRequest, since the request was valid, it's just the server is misconfigured. So I think we should technically be returning a 500 here, not a 400.